lua/music-controls/init.lua

```diff
@@ -243,7 +243,16 @@ M.shuffle = function(player)
         end
     end
 
-    exec_command('playerctl -p  '.. player[1] .. ' shuffle toggle')
+    local current_status = exec_command('playerctl -p '.. player[1] .. ' shuffle')
+    sleep(0.25)
+
+    if current_status == 'Off' then
+      exec_command('playerctl -p  '.. player[1] .. ' shuffle toggle on')
+    else
+      exec_command('playerctl -p  '.. player[1] .. ' shuffle toggle off')
+    end
```

Comment:
Please make sure to use 4 spaces as indentation.

```suggestion
    if current_status == 'Off' then
        exec_command('playerctl -p  '.. player[1] .. ' shuffle toggle on')
    else
        exec_command('playerctl -p  '.. player[1] .. ' shuffle toggle off')
    end
```

Reply:
I'm going to add a `.editorconfig` file so my auto format on save is proper for this project... I don't like having to pay attention to whether or not I've disabled it before I save.

Also, sometimes I'm not convinced tree-sitter's indent feature works correctly all the time.